pkg/ddevapp/poweroff.go

```diff
@@ -50,14 +50,17 @@ func PowerOff() {
 	if err := RemoveSSHAgentContainer(); err != nil {
 		util.Error("Failed to remove ddev-ssh-agent: %v", err)
 	}
-	// Remove current global network ("ddev") plus the former "ddev_default"
-	removals := []string{"ddev_default"}
```

Comment:
I think we should probably still remove ddev_default by name here, in case it was created without label? Open to discussion.

Reply:
This instruction to remove `ddev_default`  is below `dockerutil.RemoveNetworkWithWarningOnError(dockerutil.NetName)`